### PR TITLE
switching to using customizer, enabling maven code generators

### PR DESCRIPTION
**Pull request by KurtStam**

### CODE REVIEW

File: connectors/sql-stored-connector/pom.xml [hunk 1]
```diff
@@ -180,9 +163,11 @@
         </executions>
       </plugin>
 
-      <!-- generate components meta-data and validate component includes documentation etc
+      <!-- 
+      generate components meta-data and validate component includes documentation etc
       Disabled until the plugin supports adding parameters at the connector level. At the moment
-     they are added manually.
+      they are added manually.
```
Comment (zregvart) [inline connectors/sql-stored-connector/pom.xml]: perhaps now we can remove this comment about the plugin being disabled?

**Review by zregvart — CHANGES_REQUESTED** [review]
Just one little nit, but other that that looks good 👍

File: connectors/sql-stored-connector/pom.xml (hunk 1 above)
Comment (KurtStam) [reply connectors/sql-stored-connector/pom.xml]: haha - good find :)

**Review by zregvart — APPROVED** [review]
👍

**Pull request merged**